CHANGELOG.md

```diff
@@ -8,36 +8,88 @@
 > * [CHANGELOG_1.2x.md](./documentation/changelog/CHANGELOG_1.2x.md) - v1.20.0 to v1.29.2
 
 # UNRELEASED
-- fix(cli): handle disabled payment channel manager gracefully in lotus info command ([filecoin-project/lotus#13198](https://github.com/filecoin-project/lotus/pull/13198))
-- chore(deps): update go-libp2p to v0.42.0 ([filecoin-project/lotus#13190](https://github.com/filecoin-project/lotus/pull/13190))
```

Comment:
Maybe keep this one and combine with boxo?

> chore(deps): update of critical underlying dependencies with go-libp2p to v0.42.0 ([filecoin-project/lotus#13190](https://github.com/filecoin-project/lotus/pull/13190)) and  boxo to v0.32.0 ([filecoin-project/lotus#13202](https://github.com/filecoin-project/lotus/pull/13202))

My reason for suggesting this is just because if someone is noticing problems and looks at the changelog, I want to signal that we have done some pretty big jumps here.

Reply:
Added it in: https://github.com/filecoin-project/lotus/pull/13220/commits/96595d06a6916d3f86c87113fb0d6121404dfdb6